### PR TITLE
feat: Customize WebDriver config in custom launcher

### PR DESCRIPTION
**Pull request by tykus160**
Example of custom launcher with this PR:

```js
module.exports = function(config) {
  const firefoxOptions = {
    'moz:firefoxOptions': {
      prefs: {
        'media.eme.enabled': true,
        'media.gmp-manager.updateEnabled': true,
      },
    },
  };

  config.set({
    customLaunchers: {
      'FirefoxEME': {
        base: 'Firefox',
        config: firefoxOptions,
      },
      'FirefoxHeadlessEME': {
        base: 'FirefoxHeadless',
        config: firefoxOptions,
      },
    },
  });
};
```

### CODE REVIEW

**Comment by joeyparrish** [comment]
I think this is really useful.  But for shaka-player, I think those settings you quote in the PR description should be the defaults for Firefox.

File: index.js [hunk 1]
```diff
@@ -61,6 +62,10 @@ const LocalWebDriverBase = function(baseBrowserDecorator, logger) {
 
   log.debug('config:', JSON.stringify(config));
 
+  const extraSpecs =
+      _.merge(this.constructor.EXTRA_WEBDRIVER_SPECS, args.config);
```
Comment (joeyparrish) [inline index.js:66]: Could you please document the `config` field in the Configuration section of the README?

Also, could you please add this to the EXTRA_WEBDRIVER_SPECS for Firefox launchers by default?

```js
    'moz:firefoxOptions': {
      prefs: {
        'media.eme.enabled': true,
        'media.gmp-manager.updateEnabled': true,
      },
    },
```

**Review by joeyparrish — CHANGES_REQUESTED** [review]
Thanks!

File: index.js (hunk 1 above)
Comment (tykus160) [reply index.js:66]: Done. I hope provided example is good enough.

**Review by joeyparrish — APPROVED** [review]
Thank you!

**Pull request merged**